src/backend/chat_service/chat_service.py

```diff
@@ -133,6 +133,10 @@ async def upload_File(request: Request, customer_guid: str = Form(...), file:Upl
 async def list_files(request: Request, customer_guid: str):
     logger.debug(f"Entering list_files() with Correlation ID: {request.state.correlation_id}")
     try:
+        if not minio_manager.client.bucket_exists(customer_guid):
```

Comment:
chat_service.list_files should only call minio_manager.list_files. All checks, including bucket existence validation, should be handled within minio_manager.list_files.

If the bucket doesn't exist, ensure that an appropriate error is returned. This will help raise the correct error code and message from chat_Service.list_files.

Reply:
Okey sir